### PR TITLE
FAQ page & added email & username on discord webhook

### PR DESCRIPTION
**Pull request by tfudoinkebs**
Please test if email and username apply on 2 channels in discord webhook production-webhook & staging-webhook. Also check the contents of FAQs.

### CODE REVIEW

File: packages/constants/index.ts [hunk 1]
```diff
@@ -847,3 +848,49 @@ export const ONBOARDING_PAGES: {
     video: "https://www.facebook.com/kabsu.me/videos/755260903340962",
   },
 ];
+
+export const FAQ_ITEMS = [
+  {
+    value: "item-1",
+    question:
+      "How can freshmen create an account if we don't have a CvSU Gmail yet?",
```
Comment (bricesuazo) [inline packages/constants/index.ts]: Should be `CvSU Email` not `Gmail`

**Comment by bricesuazo** [comment]
Can we also add the FAQ section to the landing page?

**Comment by tfudoinkebs** [comment]
Added FAQ to landing page & converted jpg & png images to webp format.

image

**Review by bricesuazo — APPROVED** [review]
Nice! Thanks!

**Pull request merged**